dependencies.yaml

```diff
@@ -86,7 +86,7 @@ ext-mbedtls:
 # everest-testing and ev-dev-tools
 everest-utils:
   git: https://github.com/EVerest/everest-utils.git
-  git_tag: v0.4.2
+  git_tag: 9c7a4928251f47ffb7f8197f6cba8596fd15d242
```

Comment:
this needs to point to released version of everest-utils not a commit on a branch

Reply:
https://github.com/EVerest/everest-utils/pull/167